content/chainguard/chainguard-images/network-requirements.md

```diff
@@ -25,14 +25,15 @@ This table lists the DNS hostnames, associated ports, and protocols that will ne
 | Hostname |Port |Protocol | Notes |
 |----------|-----|---------|-------|
 | cgr.dev | 443 | HTTPS | Main image registry|
+| enforce.dev | 443 | HTTPS | Registry authentication |
```

Comment:
We should probably add the ip address list here as well, like we do in the enforce section.

Reply:
Ah great idea, will update to add those in after the table

Reply:
This look better now @strongjz? https://deploy-preview-884--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/network-requirements/